.github/workflows/webviz-subsurface-components.yml

```diff
@@ -94,6 +95,16 @@ jobs:
           npm run test_correctness --prefix ./react
           pytest ./tests --headless
 
+      # Run cypress tests on firefox browser
+      - name: E2E on firefox
+        if: github.event_name != 'release' && matrix.e2e-on-firefox
+        uses: cypress-io/github-action@v2
```

Comment:
Should we turn this into a SHA? I.e. instead of `@versionnumber` we use `@sha`

https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions